packages/plugin-core/src/workspace/templateInitializer.ts

```diff
@@ -0,0 +1,26 @@
+import { DVault } from "@dendronhq/common-all";
+import { WorkspaceService } from "@dendronhq/engine-server";
+import { BlankInitializer } from "./blankInitializer";
+import { WorkspaceInitializer } from "./workspaceInitializer";
+
+/**
+ * Template Workspace Initializer - add the templates seed to the workspace:
+ */
+export class TemplateInitializer
+  extends BlankInitializer
+  implements WorkspaceInitializer
+{
+  async onWorkspaceCreation(opts: {
```

Comment:
for the template initializer, did we also owant to open the preview to a particular note on initial open? is it worthwile creating a `onWorkspaceFirstOpen` hook?

Reply:
I had a hook, but I removed it for now - the user experience I was thinking about for this was to have templateInitializer be the default new workspace config.  It puts an empty vault (with just root.md) + this templates seed vault. So my thinking, at least at the moment, is to have the launch onWorkspaceFirstOpen experience be the same as it is now; but we can always amend this later.